pycqed/analysis/analysis_toolbox.py

```diff
@@ -1,46 +1,29 @@
-# some convenience tools
-#
 import logging
 log = logging.getLogger(__name__)
```

Comment:
Great cleanup of analysis toolbox. I would say that the next step (in a separate pull request) to reorganize the functions into separate modules (i.e. in the style of analysis_v3) to separate plotting utilities , dataloading, etc.